oktapam/client/security_policy.go

```diff
@@ -647,13 +647,9 @@ func (p *PrincipalAccountSSHPrivilege) ToResourceMap() map[string]any {
 		m[attributes.AdminLevelPermissions] = false
 	}
 	if len(p.SudoCommandBundles) > 0 {
-		scbs := make([]map[string]any, len(p.SudoCommandBundles))
```

Comment:
Updated api response mapping code. Map ID from the returned NamedObject for sudo_command_bundle.